Example/Emission/ARRootViewController.m

```diff
@@ -234,14 +237,41 @@ - (ARCellData *)jumpToUserDocs
   }];
 }
 
-- (ARCellData *)jumpToArtwork
+- (ARCellData *)jumpToArtwork1
 {
-  return [self tappableCellDataWithTitle:@"Artwork" selection:^{
+  return [self tappableCellDataWithTitle:@"Artwork1" selection:^{
```

Comment:
Can we make these names meaningful? (i.e. `Non Commercial Artwork`, `Inquiry Artwork`, `Artwork with multiple images`, etc. Feel free to choose what is most relevant or just describe the ones you've already chosen!

Reply:
We could for sure, but I was thinking to make them generic so that people can interchange a variety of ID's and not have to worry about renaming. I still think we'll be changing them in and out, I just hope not as often now. What do you think?

Reply:
I think it'll still be more helpful for our future selves to have some guidance as to what these are, even if it's rough (i.e. if I want to look at a work with multiple images, I don't want to have to remember that that's `Artwork 2` vs. `Artwork 1`).

Maybe we could keep one generic and then make the others have slightly more specific titles? I'm not too worried about the drift.